src/zicsr.adoc

```diff
@@ -168,8 +168,8 @@ a CSR, CSRW _csr, rs1_, is encoded as CSRRW _x0, csr, rs1_, while CSRWI
 _csr, uimm_, is encoded as CSRRWI _x0, csr, uimm_.
 
 Further assembler pseudoinstructions are defined to set and clear bits
-in the CSR when the old value is not required: CSRS/CSRC _csr, rs1_;
-CSRSI/CSRCI _csr, uimm_.
+in the CSR when the old value is not required: CSRS/CSRC _csr, uimm_;
```

Comment:
The last argument to csrs/csrc should be a register so rs1 was correct.

Reply:
Oh! That sentence is defining 4 pseudoinstructions. I presumed the same pattern as the previous paragraph: [pseudo] encoded as [base instruction]. My bad.

Reply:
This paragraph doesn't explicitly state what base instructions the pseudoinstructions map to. Maybe I'll add that, in case anyone else falls into the trap I did.